### DISCUSSION TITLE
Broken title when they are an emoji

### DISCUSSION THREAD

**Discussion by Enn83** [squidfunk/mkdocs-material#5889] [Q&A]
Hi!

I recently upgraded `mkdocs-material` from `9.0.15` to `9.2.3` (and `mkdocs` from `1.4.2` to `1.5.2`) and I notice that all pages where the title contain an emoji defined in Markdown way, i.e.:
```md
# Intégration et Déploiement Continu sur :simple-gitlab: Gitlab
```
... are wrongly displayed:

![262990551-07eea56f-4224-4739-b70e-97467274e8e7](https://github.com/squidfunk/mkdocs-material/assets/46783975/555a71e9-3530-4a63-906a-17617c1d0be3)

I didn't yet opened an issue since I'm not 100% sure it's not a specific bug of my documentation, but it was rendered like that before the update:

![Capture d’écran 2023-08-24 154410](https://github.com/squidfunk/mkdocs-material/assets/46783975/dcebad1a-82b1-445d-b250-8ef7aa34bafb)

So here are my questions:
* I don't know if with the recent `mkdocs-material` update (`9.2.x`) it should be displayed correctly with the emoji in the title ? 
    * ... or if the `typeset` plugin is needed (only part as Insiders) ?
* It there a way - at least - to display the title like before the update (with the emoji code)?

Thanks a lot for your hard work and thank you in advance for your reply 😃!

-----

**Edit :** I forgot to mention that I had disabled the `extra_css:` and `theme:custom_dir:` sections.
**Edit 2:** I also tried with a minimal reproductible documentation of only 1 page, and it happens too. In case here is the `Pipfile`:
```pipenv
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[requires]
python_version = "3.7"

[packages]
mkdocs = "==1.5.2"
mkdocs-material = "==9.2.3"
mkdocs-material-extensions = "==1.1.1"
mkdocs-literate-nav = "==0.6.0"
mkdocs-gen-files = "==0.5.0"
mkdocstrings = "==0.22" 
mkdocstrings-python = "==1.1.2" # More need python >= 3.8
```

**Comment by squidfunk** [comment]
Thanks for reporting! Discussions are not the place to submit bug reports. You can help us _to help you_ by opening a [new issue](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) and providing a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) (mandatory) that we can run to inspect. Might be related to #5886.
